econet_heatpump_water_heater.yaml

```diff
@@ -108,10 +108,14 @@ text_sensor:
   - platform: econet
     name: "Model Number"
     sensor_datapoint: PRODMODN
+    request_mod: 2
+    request_once: true
     icon: "mdi:information-box"
     entity_category: "diagnostic"
   - platform: econet
     name: "Serial Number"
     sensor_datapoint: PRODSERN
+    request_mod: 2
+    request_once: true
     icon: "mdi:information-box"
     entity_category: "diagnostic"
```

Comment:
Should these move to the base? Or we don't know if they work on the non-HPWH devices?

Reply:
I think @Daniel-dev22 said these don't work on HVAC.

Reply:
I think @Daniel-dev22 said these don't work on HVAC